build/AzurePipelinesTemplates/ProjectReunion-CreateNugetPackage-Job.yml

```diff
@@ -53,36 +58,52 @@ jobs:
       downloadPath: '$(Build.SourcesDirectory)\Artifacts'
 
   # debugging - remove or comment out before completing PR      
-  # - script: |
-  #     dir /s $(Build.SourcesDirectory)
+  - script: |
+      dir /s $(Build.SourcesDirectory)
 
   - template: ProjectReunion-InstallNuget-Steps.yml
 
-  - ${{if parameters.signConfig }}:
-    # Re-create framework packages because we made them in the build step against unsigned binaries.
-    - template: ProjectReunion-MakeFrameworkPackages-Steps.yml
-
-    # Copy the re-made framework packages into the same location in the artifact drop to be republished at the end of the job.
-    - task: CopyFiles@2
-      inputs:
-        targetFolder: '${{ parameters.nupkgdir }}'
-        sourceFolder: '$(Build.SourcesDirectory)\Artifacts\drop'
-        Contents: '**\Microsoft.ProjectReunion*.appx'
-
-  - powershell: |
-      $prereleaseTag = "${{ parameters.prereleaseVersionTag }}"
-      if ("${{ parameters.useReleaseTag}}" -eq [bool]::TrueString) { $prereleaseTag = "" }
+  # UNDONE to move MRTCore into Project Reunion nuget package
```

Comment:
are you going to re-enable this before completing this PR?

Reply:
No, this removes the creation of the nuget and framework packages for the Project Reunion sample app.  I'm leaving it in as a comment so we have the framework package structure in case we want to use it to build the framework packages for our new MRTCore/ProjectReunion nuget package.